language/move-bytecode-verifier/src/verifier.rs

```diff
@@ -20,6 +20,7 @@ use move_binary_format::{
 pub struct VerifierConfig {
     pub max_loop_depth: Option<usize>,
     pub treat_friend_as_private: bool,
+    pub strict_runtime_verification: bool,
```

Comment:
I agree with Wolfgang's earlier comment on naming--`strict` is the wrong word here, since it sounds like it's enforcing properties that are not enforced in non-strict mode. I would call this `redundant_runtime_verification` or `paranoid_runtime_verification` to make it very clear what the role of these checks are (a pragmatic failsafe against bugs in the verifier). This should also be explained in a comment on the config.

Reply:
I added it here just so that we don't need a new Config struct for VM.

Reply:
We will need a config struct for the VM sooner or later, so feel free to just add one
```
struct VMConfig {
  pub verifier_config: VerifierConfig,
  pub strict_runtime_verification: bool,
}
```